docs/next/components/mdx/includes/dagster/DagsterDevTabs.mdx

```diff
@@ -0,0 +1,60 @@
+<TabGroup>
+<TabItem name="From a file">
+
+Dagster can load a file directly as a code location. In the following example, we used the `-f` argument to supply the name of the file:
+
+```shell
+dagster dev -f my_file.py
+```
+
+This command loads the definitions in `my_file.py` as a code location in the current Python environment
+
+You can also include multiple files at a time:
+
+```shell
+dagster dev -f my_file.py -f my_second_file.py
```

Comment:
there is a bit of subtlety here - this will load two separate code locations, not sure if that matters here but the line above might make them think its one code location